docs/src/main/sphinx/functions/aggregate.rst

```diff
@@ -93,9 +93,29 @@ Using a filter you retain all information::
 General aggregate functions
 ---------------------------
 
+.. function:: any_value(x) -> [same as input]
+
+    Returns an arbitrary non-null value ``x``, if one exists. ``x`` can be any
+    valid expression. This allows you to return values for columns that are not
+    directly part of the aggregation in a query.
+
+    For example, the following query returns the customer name from the ``name``
+    column, and returns the sum of all total prices as customer spend. The
+    aggregation however uses the rows grouped by the customer identifier
+    ``custkey``, which is guaranteed to be unique::
```

Comment:
is the "which is guaranteed to be unique" part relevant to the function implementation?

Reply:
no .. it jus explain why the join is on custkey and not directly with the name field

Reply:
It reads a bit confusing (atleast to me) as if it's relevant for the function - but don't feel too strongly about it.